BackEndFlask/setupEnv.py

```diff
@@ -24,6 +24,46 @@ def main():
             print("[Server] attempting to run pip install requirements failed...")
             print("[Server] exiting...")
             os.abort()
+     try:
```

Comment:
While this method works, using try/catch for to cover branching logic is inefficient since it must start the incorrect installation and let it fall before continuing to the correct one. Additionally, the intent of the code is not clear since experiencing an error is still a valid state. I would highly recommend using [sys.platform](https://docs.python.org/3/library/sys.html#sys.platform) and a simple if/elif/else to immediately start the correct installation.

Reply:
I have implemented the sys.platform library and removed unclear try/catch branching logic! I have also updated exceptions to call os.abort() when an error occurs! Also I was able to remove unnecessary logic because we will no longer support Windows for development!